scripts/provision-wallet.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+WALLET_NAME=$1
+
+AGORIC_SDK=$(find ~ -type d -name "agoric-sdk" | head -n 1)
+
+WALLET_ADDR=$(agd keys show "$WALLET_NAME" --keyring-backend test --output json | jq -r .address)
```

Comment:
test?

Reply:
For now yes, we are still in the testing stage. This will change in the future.